topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -23,9 +23,8 @@ requirements:
         - scrna-case_alevin-combine-datasets
         - scrna-case_basic-pipeline
 tags:
-- single-cell
 - 10x
-- transcriptomics
+
 contributions:
```

Comment:
could add yourself to `testing`

Reply:
I figured I was already on authorship so didn't make sense to

Reply:
I think authorship + editing are mutually exclusive, one is definitely the larger contribution, but testing feels like a separate axis? At least to my feeling, it says someone sat down and tested this, and reported bugs, as a separate activity to text editing/authorship"